build.gradle

```diff
@@ -2,22 +2,25 @@ buildscript {
 	ext.kotlinVersion = '1.3.71'
 	repositories {
 		maven { url 'https://repo.spring.io/plugins-release' }
+		jcenter()
+		mavenCentral()
```

Comment:
Sorry, forgot to ask: do we really need these extra repos?

Reply:
Yes, we do. I tried it without them, but it doesn't work. Those repos are where the asciidoctor and asciidoctorPdf tasks come from, so the build breaks without them. I share your pain here: It seems goofy to add those.